src/components/Table/Table.jsx

```diff
@@ -211,6 +213,8 @@ export function Table({
 }
 
 Table.propTypes = {
+  /** Enable animation for row addition / removal. */
+  animateRows: PropTypes.oneOfType([PropTypes.bool, PropTypes.object]),
```

Comment:
Instead of overloading this prop to both enable and configure the animation, what do you think about splitting it out into two props, one to toggle animation on/off and one to configure animation? I think it might lead to a more robust design, particularly if we get into situations where we want to respect reduced motion settings for accessibility.

Reply:
Thanks for the input! I went with this approach because I really dislike adding tons and tons of props and end up with something like `Input`. Whenever I can merge 2 props into 1, I go for it 😁 (if it makes sense of course!)

It's very likely the animation will be customized somehow almost always if only to adjust the `timeout`, or get access to `onEnter` or `onExit`, so I see no reason to make folks also having to enable the thing with another prop.

And about reduced motion settings, you'll achieve that by customizing the css with `prefers-reduced-motion: no-preference`, something like:

```css
const TableUI = styled(Table)`
  .TableBody--animated {
	  .animated-row-exit {
	    background-color: transparent;
	  }
	  
	  .animated-row-exit-active {
	    background-color: red;
	    transition: background-color 500ms ease-in;
	  }
	  
	  @media screen and (prefers-reduced-motion: no-preference) {
	      .animated-row-exit {
	        background-color: transparent;
	      }
	  
	      .animated-row-exit-active {
	        background-color: red;
	        transition: none;
	      }
	   }
	}
`
```